utils/enum_option.hh

```diff
@@ -102,6 +102,9 @@ class enum_option {
     friend std::istream& operator>>(std::istream& s, enum_option<Mapper>& opt) {
         typename map_t::key_type key;
         s >> key;
+        if (key.front() == '"' && key.back() == '"') {
+            key = key.substr(1, key.size() - 2);
```

Comment:
probably i am just paranoid, but shall we check if `key.size() >= 2` ? or at least `assert(key.size() >= 2)` ?

Reply:
Certainly not assert on user input! I'll add a check.